<launch>
    <node pkg="laser_detect" type="laser_detect_node" name="laser_detect_node" output="screen">
	<param name="laser_link" value="body"/>
    </node>
</launch>
